tests/cli.rs

```diff
@@ -2707,3 +2707,40 @@ fn encoding_detection() {
     // (even for non-ASCII-compatible encodings)
     case("text/plain; charset=UTF-16", "\0\0", BINARY_SUPPRESSOR);
 }
+
+#[test]
+fn tilde_expanded_in_request_items() {
+    let homedir = TempDir::new().unwrap();
+
+    std::fs::write(homedir.path().join("secret_key.txt"), "sxemfalm.....").unwrap();
```

Comment:
Can you look for these contents in the output so we know we grabbed the right file?